src/sentry_value.c

```diff
@@ -767,6 +791,8 @@ sentry_value_is_true(sentry_value_t value)
         return 0;
     case SENTRY_VALUE_TYPE_INT32:
         return sentry_value_as_int32(value) != 0;
+    case SENTRY_VALUE_TYPE_UINT32:
+        return sentry_value_as_uint32(value) > 0;
```

Comment:
_nit_: Keep it consistent with the previous one.

```suggestion
        return sentry_value_as_uint32(value) != 0;
```